static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -361,14 +362,17 @@ const init = () => {
         ['c-public-api', initPublicApi],
         ['c-media-listeners', mediaListener],
         ['c-accessibility-prefs', initAccessibilityPreferences],
-        ['c-user-features', refreshUserFeatures],
         ['c-membership', initMembership],
         ['c-message-slots', initialiseMessageSlots],
         ['c-reader-revenue-dev-utils', initReaderRevenueDevUtils],
         ['c-add-privacy-settings-link', addPrivacySettingsLink],
         ['c-load-google-analytics', loadGoogleAnalytics],
         ['c-google-analytics', initGoogleAnalytics],
     ]);
+
+    return refreshUserFeatures().catch((err) => {
```

Comment:
Curiously it looks like `refreshUserFeatures` (or `refresh`) already returned a promise, so I guess the old error handling code never actually worked at all for this function